app/Http/Controllers/MembersController.php

```diff
@@ -221,17 +227,17 @@ public function map()
 	# Search members by coverage
 	public function search(Request $request)
 	{
-		$this->authorize("viewAny", \App\Member::class);
-		$this->authorize("showPracticalAny", \App\Member::class);
+		$this->authorize("viewAny", Member::class);
+		$this->authorize("showPracticalAny", Member::class);
 
 		$courses = $request->input('courses', []);
 		$vog = $request->input('vog', 0);
 
-		$courseList = \App\Course::orderBy('naam')->pluck('naam', 'id')->toArray();
-		$courseCodes = \App\Course::pluck('code', 'id')->toArray();
+		$courseList = Course::orderBy('naam')->pluck('naam', 'id')->toArray();
+		$courseCodes = Course::pluck('code', 'id')->toArray();
 
-		$allMembers = \App\Member::where('soort', '<>', 'oud');
-		if (Auth::user()->can("showAdministrativeAny", \App\Member::class)) {
+		$allMembers = Member::where('soort', '<>', 'oud');
+		if ($this->user->can("showAdministrativeAny", Member::class)) {
 			$allMembers = $allMembers->where('vog', '>=', $vog);
 		}
 		$allMembers = $allMembers->orderBy('voornaam')->get();
```

Comment:
I agree, more smaller services, and I quite like the Event system with the Listeners as well. But I think for this PR is just the goal of refactoring the ProfileController's methods to use the other controllers where possible. In order to prevent it from becoming a massive PR.